README.md

```diff
@@ -19,34 +19,44 @@ Before running the application, make sure you have the following installed:
 ## Installation
 
 1. Clone the repository:
-
+  
+   ```
    git clone https://github.com/samiraghav/golang-todo.git
-   <!-- Clone the repository to your local machine -->
+   ```
 
 2. Navigate to the project directory:
 
+   ```
    cd golang-todo
-   <!-- Change your working directory to the project directory -->
+   ```
 
 3. Install the dependencies:
 
+   ```
    go mod download
-   <!-- Download and install the required dependencies -->
+   ```
 
 4. Set up the MySQL database:
 
+   ```
```

Comment:
No need to mark normal sentences, just add it for code snippets

Reply:
Sure done @YashAtwal